spec/std/int_spec.cr

```diff
@@ -1,4 +1,4 @@
-require "spec"
+require "./spec_helper"
```

Comment:
Why do we need `spec_helper`?

Reply:
Good question. The code in master doesn't compile if you run just that spec file, it's missing `pending_win32` or something.

Reply:
This change shouldn't be needed anymore.

Reply:
Why not?